app/assets/javascripts/components/course_creator/scoping_methods/pagepile_scoping.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import CreatableSelect from 'react-select/creatable';
+import { UPDATE_PAGEPILE_IDS } from '../../../constants/scoping_methods';
+
+const PagePileScoping = () => {
+  const [inputValue, setInputValue] = useState('');
+  const pagePileIds = useSelector(state => state.scopingMethods.pagepile.ids);
+
+  const dispatch = useDispatch();
+
+  const handleKeyDown = async (event) => {
+    if (!inputValue) return;
+    switch (event.key) {
+      case 'Enter':
+      case 'Tab':
+        if (isNaN(inputValue)) {
+          return;
+        }
+        dispatch({
+          type: UPDATE_PAGEPILE_IDS,
+          ids: pagePileIds.concat({
+            label: inputValue,
+            value: inputValue,
+          }),
+        });
+        setInputValue('');
+        event.preventDefault();
+        break;
+      default:
+    }
+  };
+
+  return (
+    <div className="scoping-method-petscan form-group">
+      <label htmlFor="pagepile-ids">Enter PagePile IDs</label>
+      <CreatableSelect
+        inputValue={inputValue}
+        isClearable
+        isMulti
+        menuIsOpen={false}
+        onChange={ids => dispatch({ type: UPDATE_PAGEPILE_IDS, ids })}
+        onInputChange={newValue => !isNaN(newValue) && setInputValue(newValue)}
+        onKeyDown={handleKeyDown}
+        placeholder="Type something and press enter..."
```

Comment:
All these English placeholders, labels, etc, should be done with i18n... although that can be in a later PR if you don't want to implement it now.

More importantly, I think the requirement to press enter is going to cause some confusion (both in this component and probably the other ones). A user may enter a value (in this case, a PagePile ID) and then click 'Next' without first pressing enter. In that case, their input is lost but the new view is shown so the user may not realize it. (It looks like the input state is lost, as the input is empty if I click Clicking 'next' should automatically do the equivalent of pressing 'enter', if there is input that hasn't yet been converted into a multi-input value.